scdl/scdl.py

```diff
@@ -371,6 +372,7 @@ def download_playlist(playlist):
     playlist_name = playlist['title'].encode('utf-8', 'ignore')
     playlist_name = playlist_name.decode('utf8')
     playlist_name = ''.join(c for c in playlist_name if c not in invalid_chars)
+    # Sort playlist by date of track creation
```

Comment:
Is this needed here ?

Reply:
No! My bad I forgot that comment there, feel free to remove it :)